components/window/position.md

```diff
@@ -76,17 +84,27 @@ When the [Window `ContainmentSelector` parameter is set](#containmentselector),
 <TelerikWindow @bind-Left="@WindowLeft"
                @bind-Top="@WindowTop"
                Visible="true"
-               Width="300px">
+               Width="300px"
+               @ref="@WindowRef">
     <WindowTitle>Window</WindowTitle>
     <WindowContent>
-        The values of <code>WindowLeft</code> and <code>WindowTop</code> change after the user ends dragging or resizing.
+        The values of <code>WindowLeft</code> and <code>WindowTop</code> change after the user ends dragging, resizing or centering the window.
```

Comment:
The current sentence structure implies "the user ends centering the window". There is no such thing as centering as a process. Moreover, centering is programmatic and the app clears the parameter values on its own, so there is no need to mention that the parameters values will change.